v2/jam/store/disk_tmpl.go

```diff
@@ -31,7 +32,8 @@ var _ = func() error {
 }()
 `
 
-const diskImportTmpl = `// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+const diskImportTmpl = `// +build !skip-packr
```

Comment:
```suggestion
const diskImportTmpl = `// +build !skippackr
```

Reply:
Thanks for the feedback. I've updated with your changes 😄